ledger/src/blockstore_processor.rs

```diff
@@ -623,7 +623,8 @@ fn process_entries(
                     (starting_index..starting_index.saturating_add(transactions.len())).collect();
                 loop {
                     // try to lock the accounts
-                    let batch = bank.prepare_sanitized_batch(transactions);
+                    let (batch, _self_conflicting_batch) =
```

Comment:
just want to clarify, there will be a follow-up PR with adjustments in this replay scheduler in order to use the fact its' a self-conflicting entry.

As PR currently stands it doesn't seem to satisfy the 2nd item of #1025

Reply:
yes, you are right, the PR that handles the conflicting entries on replay side is also in progress.